lib/src/models/transaction_chunk.dart

```diff
@@ -35,11 +35,11 @@ class TransactionOffsetResponse {
 
 @JsonSerializable()
 class TransactionChunkResponse {
-  final String chunk;
+  final String? chunk;
```

Comment:
Why allow these to be optional?

Reply:
I think in general when you allow things like this to be optional you should leave a comment behind about why they can be optional or what nullness would mean.